com.unity.netcode.gameobjects/Runtime/Transports/UTP/SecureAccessor.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.IO;
+using UnityEngine;
+
+namespace Unity.Netcode.Transports.UTP
+{
+    public class SecureAccessor : MonoBehaviour
```

Comment:
Minor, but this perhaps could have a better name, since by the name alone I'm not sure what it does. Maybe something like `SecretsLoader` or something along those lines?

Reply:
Also I guess it should have some XML documentation comments. Otherwise it will trigger warnings in the package promotion tests. Not a deal breaker, but since there was an effort lately to reduce these warnings in NGO, I guess it would be best to avoid adding too many new ones.

Reply:
Renamed, and, yes, will add XML doc